source/administration/command-line-tools.rst

```diff
@@ -364,7 +365,57 @@ mattermost command
     Commands for slash command management.
 
   Child Commands
-    -  `mattermost command move`_ - Move a slash command to a different team
+    -  `mattermost command create`_ - Create a custom slash command for a specified team.
+    -  `mattermost command delete`_ - Delete a slash command.
+    -  `mattermost command move`_ - Move a slash command to a different team.
+    -  `mattermost command list`_ - List all commands on specified teams or all teams by default.
+
+mattermost command create 
+~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Create a custom slash command for a specified team. 
+
+  Format
+    .. code-block:: none
+
+      mattermost command create 
+
+  Examples
+    .. code-block:: none
+
+      sudo ./mattermost command create myteam --title MyCommand --description "My Command Description" --trigger-word mycommand --url http://localhost:8000/my-slash-handler --creator myusername --response-username my-bot-username --icon http://localhost:8000/my-slash-handler-bot-icon.png --autocomplete --post
+
+  Options
+    .. code-block:: none
+
+          --title string  Command Title
```

Comment:
Propose vertically aligning the command and the description so that it's easier to scan through the options, similar to attached below

![image](https://user-images.githubusercontent.com/13119842/48854170-3758dc00-ed7f-11e8-9f40-649ad00eb477.png)

Reply:
Thanks - updated a few other locations as well.